src/components/Avatar/Avatar.jsx

```diff
@@ -125,7 +125,7 @@ Avatar.propTypes = {
   backgroundColor: PropTypes.oneOf(Object.values(Avatar.colors)),
   role: PropTypes.string,
   ariaLabel: PropTypes.string,
-  size: PropTypes.oneOf([Avatar.sizes.LARGE, Avatar.sizes.MEDIUM, Avatar.sizes.SMALL]),
+  size: PropTypes.oneOf([Avatar.sizes.LARGE, Avatar.sizes.MEDIUM, Avatar.sizes.SMALL, Avatar.sizes.XS]),
```

Comment:
instead of set a new fixed size let's add support on new prop inside avatar which called: "size" and it received a number.
if avatar has size prop lets set the size of the avatar according to the pixels inside the prop instead of by the avatar.size prop